src/main/bash/commands/kobman-uninstall.sh

```diff
@@ -13,41 +13,41 @@ if [[ $environment == "all" && -z $version ]]; then
   __kobman_echo_white "This operation would remove all the environments and its files"
   __kobman_interactive_uninstall || return 1
   __kobman_echo_white "Removing files..."
-  rm -rf $KOBMAN_DIR/envs/kob_env_*
-  rm -rf ~/Dev_*
+  find $KOBMAN_DIR/envs -maxdepth 1 -mindepth 1 -type d -exec rm -rf '{}' \;
+  rm -rf $HOME/Dev_*
```

Comment:
The user or applications may not follow a naming convention like Dev_ when it comes to selecting a name for the root folder where the environment should be hosted. So best to avoid Dev_ prefix .Can't it be local variable like kobman_env_root_folder?